forest/src/daemon.rs

```diff
@@ -42,6 +44,17 @@ use forest_fil_cns::composition as cns;
 #[cfg(feature = "forest_deleg_cns")]
 use forest_deleg_cns::composition as cns;
 
+fn set_event() {
```

Comment:
Maybe rename to `signal_event` and describe why this function is used and what happens if we didn't detach (ie. that the event is still valid but no parent will be listening for the signals, etc.)

Reply:
Could even name it `unblock_parent_process`.

Reply:
Note if we don't use `--detach` the pointer is still null and we just bail out (and never call `set`).

Reply:
Great. Add that to the documentation.